src/spu/src/smart_stream/mod.rs

```diff
@@ -1,92 +1,11 @@
-use std::path::Path;
-use std::sync::{RwLock, Mutex};
-
-use anyhow::{Result, Error};
-use tracing::{debug, instrument};
-use wasmtime::{Engine, Module, Store, Instance, Memory};
-use crate::smart_stream::filter::SmartFilter;
+use std::sync::Mutex;
+use anyhow::Result;
+use wasmtime::{Memory, Store};
 
 mod memory;
 pub mod filter;
 pub mod file_batch;
 
-pub struct SmartStreamEngine(Engine);
```

Comment:
Prefer to keep abstraction instead of exposing it directly

Reply:
I can revert this but I think it won't do anything other than add noise where it is used, since the wasmtime API calls require `&Engine` directly.